odoo/addons/test_assetsbundle/tests/test_assetsbundle.py

```diff
@@ -42,12 +42,15 @@ def _get_asset(self, xmlid, env=None):
         files, remains = env['ir.qweb']._get_asset_content(xmlid, env.context)
         return AssetsBundle(xmlid, files, env=env)
 
-    def _any_ira_for_bundle(self, type, lang=None):
+    def _any_ira_for_bundle(self, type, lang=None, is_minified=False, root=False):
         """ Returns all ir.attachments associated to a bundle, regardless of the verion.
         """
         user_direction = self.env['res.lang']._lang_get(lang or self.env.user.lang).direction
         bundle = self.jsbundle_xmlid if type == 'js' else self.cssbundle_xmlid
-        url = '/web/content/%-%/{0}{1}.{2}'.format(('rtl/' if type == 'css' and user_direction == 'rtl' else ''), bundle, type)
+        if root:
+            url = '/{0}{1}.{2}'.format(('rtl/' if type == 'css' and user_direction == 'rtl' else ''), bundle, "min.%s" % type if is_minified else type)
+        else:
+            url = '/web/content/%-%/{0}{1}.{2}'.format(('rtl/' if type == 'css' and user_direction == 'rtl' else ''), bundle, "min.%s" % type if is_minified else type)
```

Comment:
Indices are not actually useful. Also maybe define `direction` and `extension` local variables & use f-strings since you're using that elsewhere?